drawer/drawer.css

```diff
@@ -0,0 +1,103 @@
+@import url(/style.css);
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-size: 62.5%;
+}
+
+:root {
+
+    --RED: #ff0000;
+    --RED-50: #f75353;
+    --RED-20: #f2dede;
+    --GREEN: #01aa01;
+    --GREEN-50: #51f751d2;
+    --GREEN-20: #dff0d8;
+    --ORANGE: #ce8602;
+    --ORANGE-50: #f7bb4c;
+    --ORANGE-20: #fcf8e3;
+    --BLUE: #0000ff7e;
+    --BLUE-50: #5353f1;
+    --BLUE-80: #3d3df3;
+
```

Comment:
You already have these variables declared in style which you are importing, You don't need to declare them again